wp-content/themes/missilethreat/assets/_scss/layout/_footer.scss

```
@@ -1 +1,103 @@
 @use 'abstracts' as *;
+
+.footer {
+  @extend %font-body-short-md;
+  @include full-width-background($color-primary-blue-100);
+  padding-top: rem(40px);
+  padding-bottom: rem(16px);
+  color: $color-text-light-170;
+  background-color: $color-primary-blue-100;
+}
+
+.footer__contact {
+  display: flex;
+  flex-wrap: nowrap;
+  margin-top: rem(32px);
+}
+
+.footer_address {
+  margin-left: rem(32px)
+}
+
+.footer__logo, .footer_mdp, .footer_pp {
+  color: $color-text-light-100;
+}
+
+.footer_nav_side {
+  color: $color-text-light-190;
+}
+
+.footer__copyright {
+  color: $color-text-light-155;
+}
+
+.footer__logo {
+  margin-bottom: rem(24px);
+}
+
+.footer_social_icons {
+  display: flex;
+  align-items: center;
+}
+
+.facebook_icon, .twitter_icon {
+  padding: rem(8px) rem(9px);
+  border-radius: rem(50px);
+  background-color: $color-text-light-100;
+  color: $color-primary-blue-100;
+  vertical-align: middle;
+}
```

This is still a little off.  Try something like this
```suggestion
.facebook_icon, .twitter_icon {
  padding: rem(6.5px) rem(9px);
  border-radius: rem(50px);
  background-color: $color-text-light-100;
  color: $color-primary-blue-100;
}

.icon {
  vertical-align: middle;
}
```